### PR TITLE
yak-shave update

### PR DESCRIPTION
**Pull request by aahunter**

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -15,18 +15,29 @@ env | grep -iv PASS | sort
...
     echo "[Run] developing site"
+
     gulp_run
-    cd /app/site && exec bundle exec jekyll serve --host=0.0.0.0
+
+    cd /app/site 
+    bundle 
+
+    exec jekyll serve --host=0.0.0.0
 fi
```
Comment (grahame) [inline docker-entrypoint.sh:23]: typo, I think? shouldn't bundle ... exec be all on one line?

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -15,18 +15,29 @@ env | grep -iv PASS | sort
...
     echo "[Run] building production tarball"
+
     gulp_run
-    cd /app/site && bundle exec jekyll build
-    cd /app/site/_site && exec tar czf /build/${TARBALL_NAME}-${TARBALL_GIT_TAG}.tar.gz .
+
+    cd /app/site
+    bundle
+    jekyll build
+
```
Comment (grahame) [inline docker-entrypoint.sh:32]: same thing here, `bundle jekyll build` should be one line (and the exec should stay in there, it's an argument to `bundle` not a shell `exec`)

**Review by grahame — CHANGES_REQUESTED** [review]

File: docker-entrypoint.sh (hunk 2 above)
Comment (aahunter) [reply docker-entrypoint.sh:32]: sorry my mistake

**Review by grahame — APPROVED** [review]
looks good

**Pull request merged**